Final Project App/Final Project App/ViewController.swift

```diff
@@ -133,30 +123,27 @@ extension ViewController: UITextFieldDelegate {
     
     // Send data to server and clear textfield when return is pressed 
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
-        if let newText = InputTextField.text {
-            let newInput: Parameters = ["text" : newText, "username" : "Eric"]
-            Alamofire.request("http://localhost:1337/messages", method: .post, parameters: newInput)
+        if let newText = inputTextField.text {
+            let newInput: Parameters = ["text" : newText, "username" : currentUser as Any]
+            Alamofire.request(chatServer, method: .post, parameters: newInput)
         }
-        InputTextField.text = ""
+        inputTextField.text = ""
         return true
     }
     
 }
 
 //MARK: - Hide Keyboard on tap outside of textfield
-extension UIViewController
-{
-    func hideKeyboard()
-    {
+extension UIViewController {
+    func hideKeyboard() {
```

Comment:
This is why you don't name things badly. I thought this function hid the keyboard. `hideKeyboard` does not hide the keyboard. It should be something like `installDismissKeyboardGestureRecognizer`